UI-tests/pom.xml

```diff
@@ -7,7 +7,7 @@
 	<parent>
 		<groupId>com.capitalone.dashboard</groupId>
 		<artifactId>Hygieia</artifactId>
-		<version>3.0.1</version>
+		<version>3.0.2-SNAPSHOT</version>
```

Comment:
@shriver135  can you create property for version in Hygieia/pom.xml and use it across.

in  <properties>
...
</properties>